### DISCUSSION TITLE
Way to override some binds/settings in fzf

### DISCUSSION THREAD

**Discussion by UnkwUsr** [zk-org/zk#168] [Ideas]
Usually I scroll fzf with `ctrl-n`/`ctrl-p`, but `zk edit -i` opens fzf with `ctrl-n` binded to creating a new note, so I'm very often forget about it, and in result I have tons of empty notes.

For now I see at least one easy solution: add config setting like `fzf-opts`. Then I will can write something like:

    fzf-opts = "--bind ctrl-n:down"

And it will override previously defined keybind.

Also this setting can be useful for other scenarios.

**Comment by mickael-menu** [comment]
I changed the default key binding to `Ctrl-E` to avoid conflicts with `Ctrl-N`, but now you can also customize (or remove) it with [the new `fzf-bind-new` config option](https://github.com/mickael-menu/zk/blob/main/docs/tool-fzf.md#key-bindings).

```toml
[tool]
fzf-bind-new = "Ctrl-C"

**Reply by UnkwUsr** [reply]
Thanks!
